sippy-ng/src/component_readiness/CompReadyTestReport.js

```diff
@@ -304,8 +305,13 @@ export default function CompReadyTestReport(props) {
           &gt; {testName}{' '}
         </Link>
       </h3>
-      Test Name: {testName}
-      <hr />
+      <div align="center" style={{ marginTop: 50 }}>
+        <h2>{testName}</h2>
+      </div>
+      <h2>Known Bugs</h2>
```

Comment:
Does it make it clearer if this is center aligned?

Reply:
I don't think so, section headers are left-aligned in Sippy (i.e. [Test Analysis](https://sippy.dptools.openshift.org/sippy-ng/tests/4.15/analysis?test=OSD%20e2e%20suite.%5BSuite%3A%20operators%5D%20%5BOSD%5D%20Must%20Gather%20Operator%20clusterRoles%20should%20exist&filters=%7B%22items%22%3A%5B%7B%22columnField%22%3A%22name%22%2C%22operatorValue%22%3A%22equals%22%2C%22value%22%3A%22OSD%20e2e%20suite.%5BSuite%3A%20operators%5D%20%5BOSD%5D%20Must%20Gather%20Operator%20clusterRoles%20should%20exist%22%7D%2C%7B%22columnField%22%3A%22variants%22%2C%22not%22%3Atrue%2C%22operatorValue%22%3A%22contains%22%2C%22value%22%3A%22never-stable%22%7D%2C%7B%22columnField%22%3A%22variants%22%2C%22not%22%3Atrue%2C%22operatorValue%22%3A%22contains%22%2C%22value%22%3A%22aggregated%22%7D%5D%2C%22linkOperator%22%3A%22and%22%7D)).  CR isn't exactly homogenous with the rest but I'd prefer to stay consistent.